src/pages/token/rewards/RewardsPage.tsx

```diff
@@ -29,111 +31,136 @@ const RewardsPage = () => {
   const { isTablet, isNotTablet } = useBreakpoints();
   const navigate = useNavigate();
 
-  const showChartPanel = testFlags.tradingRewardsRehaul;
+  const tradingRewardsRehaulEnabled = testFlags.tradingRewardsRehaul;
 
-  return (
+  const legalDisclaimer = (
+    <$LegalDisclaimer>
+      {stringGetter({ key: STRING_KEYS.TRADING_REWARDS_LEGAL_DISCLAIMER })}
+    </$LegalDisclaimer>
+  );
+  return isTablet ? (
```

Comment:
nit: for mobile view, the table can now overflow horizontally whereas it couldn't previously, can we fix that in css back?
<img width="995" alt="Screenshot 2024-05-31 at 1 33 17 PM" src="https://github.com/dydxprotocol/v4-web/assets/9400120/2aef989a-1a31-46ad-86b2-830be246e667">

Reply:
yess fixed!